trello/features/functional/BoardFunctional.feature

```diff
@@ -1,6 +1,14 @@
 # Created by Enrique Carrizales at 1/29/2020
-Feature: #Enter feature name here
-  # Enter feature description here
+Feature: Board's Functional Test
+  As a regular user, Manages the board, and user creates a new label of board.
 
-  Scenario: # Enter scenario name here
-    # Enter steps here
\ No newline at end of file
+  # Created by Oscar Lopez at 1/29/2020
+  Scenario: Creates new label at board with a color
```

Comment:
It would be creates a new Board with label

Reply:
This is like a update board

Reply:
Ok, that's right